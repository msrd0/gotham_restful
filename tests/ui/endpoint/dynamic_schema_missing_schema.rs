#[macro_use]
extern crate gotham_restful;

use gotham::hyper::StatusCode;

#[derive(Resource)]
#[resource(read_all)]
struct FooResource;

fn status_codes() -> Vec<StatusCode> {
	unimplemented!()
}

#[read_all(status_codes = "status_codes")]
async fn read_all() {}

fn main() {}
