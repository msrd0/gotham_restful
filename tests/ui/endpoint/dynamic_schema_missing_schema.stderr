error: `schema` and `status_codes` may only be used together
  --> tests/ui/endpoint/dynamic_schema_missing_schema.rs:14:27
   |
14 | #[read_all(status_codes = "status_codes")]
   |                           ^^^^^^^^^^^^^^

error[E0412]: cannot find type `read_all___gotham_restful_endpoint` in this scope
 --> tests/ui/endpoint/dynamic_schema_missing_schema.rs:7:12
  |
7 | #[resource(read_all)]
  |            ^^^^^^^^ not found in this scope
